pkg/okteto/client.go

```diff
@@ -344,6 +344,8 @@ func translateAPIErr(err error) error {
 		return fmt.Errorf("server temporarily unavailable, please try again")
 	case "non-200 OK status code: 401 Unauthorized body: \"\"":
 		return fmt.Errorf("unauthorized. Please run 'okteto context url' and try again")
+	case "non-200 OK status code: 401 Unauthorized body: \"not-authorized\\n\"":
```

Comment:
this change makes the CLI to return a meaningful error, otherwise we would always return internal server error for invalid tokens

Reply:
I'm still getting the internal server error. I'm running a `okteto deploy`  with a token that doesn't exists and I'm getting a different GraphQL response:
`non-200 OK status code: 401 Unauthorized body: "fail to find user with token XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX30ZJ: not-authorized\n"
Is this ok?

Reply:
This is the case I mentioned in my comment

Reply:
thanks guys, handled in b039f9b9c86433b3f4703fba4f58d7b885901d5b